### PR TITLE
ticket 13 help needed

### PR DESCRIPTION
**Pull request by farwah1**

### CODE REVIEW

File: models/topics.js [hunk 1]
```diff
@@ -128,5 +129,10 @@ exports.selectUsers = () => {
...
 
-
-
+exports.selectApi  = async() => {
+    return await readFile('./endpoints.json', 'utf8')
+    .then((endpoints) => {
+        const parsedEndpoints = JSON.parse(endpoints)
+        return JSON.stringify(parsedEndpoints)
+    })
+}
```
Comment (xViolaine) [inline models/topics.js]: For this, we should actually not need the models at all!

**Review by xViolaine — CHANGES_REQUESTED** [review]

File: models/topics.js (hunk 1 above)
Comment (farwah1) [reply models/topics.js]: removed this function from models!

File: endpoints.json [hunk 2]
```diff
@@ -23,5 +25,78 @@
...
+  "GET /api/users": {
+    "description": "serves an array of all users",
+    "exampleResponse": {
+      "users" :
+      [{
+        "username": "butter_bridge",
+        "name": "jonny",
+        "avatar_url": "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+      }]
+    }
```
Comment (Rayhaan2k9) [inline endpoints.json:101]: Don't forget the DELETE endpoint!

**Review by Rayhaan2k9 — APPROVED** [review]
Please add delete endpoint to the JSON file and then merge away! 😄

**Pull request merged**